### PR TITLE
Extended components props

### PR DESCRIPTION
**Pull request by Aeonax**
Improving ... custom styles access area?
And some possible improvements that noticed during work/eventum-ui investigations.

### CODE REVIEW

File: src/Buttons.js [hunk 1]
```diff
@@ -168,18 +169,20 @@ export function Button({
   children,
   disabled,
   onClick,
-  style,
+  className,
+  ...orherProps
```
Comment (bskog) [inline src/Buttons.js]: Typo? "orherProps" -> "otherProps"

File: src/DateSelectButton.js [hunk 2]
```diff
@@ -24,13 +25,14 @@ const i18n = {
...
-export function DateSelectButton({ date, label, locale, onClick }: EventDateButtonProps) {
+export function DateSelectButton({ date, label, locale, onClick, wrapperProps }: EventDateButtonProps) {
   const text = i18n[locale];
+  const WrapperProps = wrapperProps || {};
 
   dateformat.masks.shortDate = text.shortDateFormatMask;
 
   return (
-    <div>
+    <div {...WrapperProps}>
```
Comment (bskog) [inline src/DateSelectButton.js]: Don't capitalize. And I think it's safe to do `{...wrapperProps}` directly.
What is the use case for the `wrapperProps`? Anything else than `className` or `style` you need on that `div`?

Comment (Aeonax) [reply src/DateSelectButton.js]: I got errors when `wrapperProps` was `null`, I even tried `...(wrapperProps || {})`, that worked in console, but still got errors when "opened" it in component...

Hmmm... yep, the main purpose was for style and class...

Comment (Aeonax) [reply src/DateSelectButton.js]: Appeared `Cannot read property 'className' of undefined` when removing that crunch...

But maybe it really would be better to pass wrapperStyle || wrapperClass...=)

File: src/FormComponents.js [hunk 3]
```diff
@@ -6,14 +6,14 @@ type LabelProps = {
...
+  wrapperStyle?: Object,
   otherProps?: Object,
 }
 
-export function Label({ description, htmlFor, label, wrapperProps, ...otherProps }: LabelProps) {
-  const WrapperProps = wrapperProps || {};
+export function Label({ description, htmlFor, label, wrapperClass, wrapperStyle, ...otherProps }: LabelProps) {
   return (
-    <div className="mod" {...WrapperProps}>
+    <div className={wrapperClass || "mod"} style={wrapperStyle}>
```
Comment (bskog) [inline src/FormComponents.js]: `wrapperClass` should add to current style, not overwrite. So: `className={'mod ' + wrapperClass ? wrapperClass : ''}`

File: src/Grid.js [hunk 4]
```diff
@@ -4,12 +4,12 @@ import * as React from 'react';
...
   flex?: number | string,
-  style?: Object,
+  otherProps?: Object,
 }
 
-export function Col({ children, flex, style }: ColProps) {
+export function Col({ children, flex, ...otherProps }: ColProps) {
   return (
-    <div style={style}>
+    <div {...otherProps}>
```
Comment (bskog) [inline src/Grid.js]: `className` and `style` prior to `otherProps` also here?

**Pull request merged**